### PR TITLE
Feat/ctc 37/update subject listing query unit mv

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

update subject listing mv to use published_mv_synthetic_unitvariants_with_lesson_ids_by_keystage_14_0_1

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-3033--oak-web-application.netlify.thenational.academy
2. Click on subject listing
3. You should see no regressions

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const HomePageTabImageNav = ({ current }: { current: HomePageTab }) => {
           isActive={current === "curriculum"}
         />
         <OakHomepageTabButton
-          title="AI Experiments"
+          title="AI experiments"
```
Comment (kimon-satan) [inline src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx:36]: How did this end up here ?

File: src/node-lib/curriculum-api-2023/queries/lessonDownloads/rawSyntheticUVLesson.schema.ts [hunk 2]
```diff
@@ -1,8 +1,5 @@
 import { z } from "zod";
-import {
-  syntheticUnitvariantLessonsByKsSchemaOld,
-  syntheticUnitvariantLessonsSchema,
-} from "@oaknational/oak-curriculum-schema";
+import { syntheticUnitvariantLessonsSchema } from "@oaknational/oak-curriculum-schema";
```
Comment (kimon-satan) [inline src/node-lib/curriculum-api-2023/queries/lessonDownloads/rawSyntheticUVLesson.schema.ts:2]: hmm why is this file modified for the subject listing query ?

**Review by RBrownsett — APPROVED** [review]
Approved - subject pages checked for all Key Stages, no issues. Full regression will be carried out on the feature branch

File: src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx (hunk 1 above)
Comment (benprotheroe) [reply src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx:36]: refresh
